tests/func/test_run_single_stage.py

```diff
@@ -204,22 +204,30 @@ def test_non_normalized_paths(self):
 class TestRunStageInsideOutput(TestDvc):
     def test_cwd(self):
         self.dvc.run(
-            cmd="", deps=[], outs=[self.DATA_DIR], single_stage=True,
+            cmd=f"mkdir {self.DATA_DIR}",
```

Comment:
There should not too many of the cases were there are no deps but only outs, but in those cases I have used `mkdir` or `touch` as `cmd`.